tools/block-generator/generator/generate_apps.go

```diff
@@ -0,0 +1,260 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
```

Comment:
Most of the code here was migrated from `generator.go`. I've left comments on their previous location describing any changes. I'm also commenting new functions here.